setup/openchange_provision

```diff
@@ -29,6 +29,24 @@ import samba
 import samba.getopt as options
 import openchange.provision as openchange
 
+def check_incompatible_options(parser, opts):
```

Comment:
Why don't you use callbacks?
https://docs.python.org/2/library/optparse.html#callback-example-4-check-arbitrary-condition